gulpfile.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable */
+const gulp = require('gulp')
```

Comment:
One for us in the future later, after some research I think there could be an easy way for us to deal with this simply using jest.
Jest exposes a global hook to run something after (and/or) before our actual tests run, so it could be the case that we can just use that and get rid of that.
The documentation is here: https://jestjs.io/docs/26.x/configuration#globalsetup-string

I think we have a ticket in the backlog for this too: https://jira.dvsacloud.uk/browse/CVSB-19406

Reply:
This will fail in the pipeline and not needed anymore we can remove the file. I am working on preparers to have the script having similar behaviours in the pipeline as we have locally
https://github.com/dvsa/cvs-svc-preparers/pull/45/files#diff-8f5ecaff97ebbcd15a37d148a98765ecd8e3b97a86455254444b915f19f0df08R1
At the moment it works locally but it seems that the docker container that runs inside a docker container have pid managed differently and the server is not exiting gracefully once the tests are ran.